Examples/Io/Root/src/RootMaterialTrackReader.cpp

```diff
@@ -90,23 +87,19 @@ ActsExamples::RootMaterialTrackReader::RootMaterialTrackReader(
             << " events this corresponds to a batch size of: " << m_batchSize
             << std::endl;
 
-  // If the events are not in order, get the entry numbers for ordered events
-  if (!m_cfg.orderedEvents) {
-    if (!eventIdPresent) {
-      throw std::invalid_argument{
-          "'event_id' branch is missing in your tree. This is not compatible "
-          "with unordered events."};
-    }
+  // If the events are not ordered, we need to sort the entry numbers
```

Comment:
I think we always order them, even when they are sorted?
I like the scope tough, maybe comment as
```suggestion
  // Sort the entry numbers of the events
```